gradle.properties

```diff
@@ -3,3 +3,4 @@ buildSystemRootDir=vividus-build-system
 
 org.gradle.parallel=true
 org.gradle.caching=true
+org.gradle.jvmargs=-Xmx2g
```

Comment:
looks like a workaround :)

Reply:
that's true, I assume there is some issue with SpotBugs, but it's too expensive to debug it